training/general/src/environments/acpt-pytorch-2.0-cuda11.7/tests/ipex2_0_sample_text.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.

```

Comment:
Can you correct the file name? It says "ipex2_0_sample_text.py" Shouldn't it be "ipex2_0_sample_test.py"?